app/test_split_integer.py

```diff
@@ -2,20 +2,36 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 15
+    number_of_parts = 3
```

Comment:
Do not create additional variables, pass values directly to the function and make required actions in the `assert`.